### ISSUE TITLE
elpy goto test?

### ISSUE THREAD

**Issue by ChillarAnand** [jorgenschaefer/elpy#507]
Elpy has`elpy-goto-definition` which quickly jumps to definition. In the same way, if we have `elpy-goto-test`, so that we can switch back/forth between functions & its tests it will be very effificient for TDD(or to run tests in general).

**Comment by jorgenschaefer** [comment]
Good idea, sadly a bit tricky, as it's not simple to identify the location for a test for something at point.

If you want, you can install projectile and use `C-c p t`, which tries to implement that feature. I'm thinking of making Elpy depend on projectile, so if you do that, let me know if that does what you are looking for?

**Comment by ChillarAnand** [comment]
i checked projectile toggle command today & thought about this.

There are several commonly accepted places to put tests for a module:

In the same directory as module.py.
In ../tests/test_module.py (at the same level as the code directory).
In tests/test_module.py (one level under the code directory).

so we can check for a function/class which is appended or prepended with  `test`?


projectile integration will be awesome :)

**Comment by ChillarAnand** [comment]
there is  one more issue with projectile test toggle:

if i am working with django & want to run tests for `views.py` for an app, if i do `C-c p t`, it shows __all__ `test_views.py` files from all apps

**Comment by jorgenschaefer** [comment]
So in general, projectile's implementation is "good enough", but there are some rough edges that should be straightened out for the integration. Good to know, thank you!
